package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3007",
```

Comment:
What's taking the default port? If you need custom confs, I'd suggest running the command directly for yourself, without changing the "npm run" command

Reply:
> What's taking the default port? If you need custom confs, I'd suggest running the command directly for yourself, without changing the "npm run" command in the repo

Sure, you right here, i was changing it just for local.